backend/pom.xml

```diff
@@ -60,7 +65,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
-            <plugin>
+            <!--plugin>
```

Comment:
This plugin is quite important for the running of the react framework. I am curious as to why it is being commented out? Are there issues associated with the plugin?

Reply:
Will discuss on Discord